docs/sources/twilio.md

```diff
@@ -3,11 +3,7 @@ This event source is to be deployed and then registered as a webhook via Twilio
 
 ## Prerequisite(s)
 
-* Twilio Account
```

Comment:
Was this meant to be part of this PR?

Reply:
I was going to change `*` to `-` in the same pull request. However, I think the census was that I should create a new Issue and Pull Request to fix that. I can do it in the Pull Request if it makes it easier.

Reply:
Yes I also think each PR should have a well defined scope. It probably makes more sense to address all those little changes in their own PR. Besides the "Changed files" tab shows more than just a change from `*` to `-`.